concent_api/core/payments/bankster.py

```diff
@@ -373,20 +371,11 @@ def settle_overdue_acceptances(
 
             # Concent checks if all passed SubtaskResultAccepted messages from subtask_results_accepted_list
             # have payment_ts < T1
-            t1_is_bigger_than_payments_ts = any(
+            if any(
```

Comment:
If I understand correctly, this is protection against double spending? If so, exception name seems wrong (and the returned message in the exception handler)

Reply:
If you are adding a comment that something is wrong it would be good to write what should be the correct version to avoid circulation.